contrib/kafka/main.go

```diff
@@ -2,14 +2,11 @@ package main
 
 import (
 	"flag"
-	"fmt"
-	"os"
-	"strings"
-
 	istiov1alpha3 "github.com/knative/pkg/apis/istio/v1alpha3"
 	"go.uber.org/zap"
 	"k8s.io/apimachinery/pkg/runtime"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"os"
```

Comment:
Did you run `gofmt` on this file? I'd have expected it to put this import above next to `flag`.

Reply:
hmm  I did run `gofmt -w main.go`, nothing changes, then I purposely put next to `flag` and rerun `gofmt` it puts `os` back after `_ k8s.io/client-go/plugin/pkg/client/auth/gcp`, looks weird. I am using `go version go1.11.5 darwin/amd64`

Reply:
weird! I don't know why it does that, but I guess it's correct by definition. So :+1: